linux/ubuntu/scripts/basic.sh

```diff
@@ -5,12 +5,21 @@
 ################################################################################
 # source: https://github.com/actions/virtual-environments/blob/5ae2170ebe90a53e1cdc9c507ed3e0f1471d6b66/images/linux/scripts/helpers/install.sh
 
+. /etc/os-release
+
 apt-get update
 # lib32z1 is not available for arm64 remove it via jq
 case "$(uname -m)" in
   'x86_64') common_packages_filter="" ;;
   *) common_packages_filter="del(.apt.common_packages[] | select(. == \"lib32z1\"))" ;;
 esac
+case "${VERSION_ID}" in
+  '24.04')
+    jq '.apt.cmd_packages |= map(if . == "netcat" then "netcat-openbsd" else . end)' "/imagegeneration/toolset.json" > "/tmp/toolset.new.json" \
+      && mv "/tmp/toolset.new.json" "/imagegeneration/toolset.json"
+    ;;
+  *) ;;
+esac
```

Comment:
I think all Ubuntu releases onward have this new package name:

```suggestion
UBUNTU_MAJOR=${VERSION_ID%%.*}
if (( UBUNTU_MAJOR >= 24 )); then
  jq '.apt.cmd_packages |= map(if . == "netcat" then "netcat-openbsd" else . end)' "/imagegeneration/toolset.json" > "/tmp/toolset.new.json" \
    && mv "/tmp/toolset.new.json" "/imagegeneration/toolset.json"
fi
```

Reply:
Unfortunately GitHub still has bugs, even when I select the correct lines 🤷

```shell
UBUNTU_MAJOR=${VERSION_ID%%.*}
if (( UBUNTU_MAJOR >= 24 )); then
  jq '.apt.cmd_packages |= map(if . == "netcat" then "netcat-openbsd" else . end)' "/imagegeneration/toolset.json" > "/tmp/toolset.new.json" \
    && mv "/tmp/toolset.new.json" "/imagegeneration/toolset.json"
fi
```

Reply:
If you want to, yes, then Ubuntu 25+ would be working too 😄